nixos/modules/services/misc/bazarr.nix

```diff
@@ -0,0 +1,80 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bazarr;
+in
+{
+  options = {
+    services.bazarr = {
+      enable = mkEnableOption "bazarr";
```

Comment:
Personal pet peeve: please add a _bit_ info here, like instead of just "bazarr", please change to "bazarr, a subtitle manager for Sonarr and Radarr".